src/components/Episodes/renderUtils.jsx

```diff
@@ -1,29 +1,40 @@
 import React from "react";
-import Episode from "COMPONENTS/Episode/";
+import Episode, { TrailerProvider } from "COMPONENTS/Episode/";
+import { data as episodesData } from "./";
+import UTILS from "SHARED/utils.js";
 
 function renderEpisode(data, index){
 
 	const { 
 		number,
 		title,
 		synopsis,
-		image,
-		video,
+		thumbnail,
+		trailer,
 	} = data;
 
-	const id = `episode__${number}`;
+	const id = UTILS.convertToSafeString(title, "-");
+
+	const { trailer: prevTrailer } = episodesData[index - 1] || {};
+	const { trailer: nextTrailer } = episodesData[index + 1] || {};
 
 	return (
-		<Episode
-			id={ id }
-			key={ id }
-			index={ index }
-			number={ number }
-			title={ title }
-			synopsis={ synopsis }
-			image={ image }
-			video={ video }
-		/>
+		<TrailerProvider 
+			prev={ prevTrailer }
+			next={ nextTrailer } 
+			key={ `${id}-trailer-provider` }
```

Comment:
BEM! `provider__trailer__${id}`

Reply:
well, most bem-iest bem would be `provider__trailer--${id}` but I tend to save that for CSS only - I keep to either `_` or `-` for ids/keys/classes, with my arbitrary rule being that if it's for internal logic, use `_` and if the user's going to see it then use `-`.